Automattic/wp-calypso
# Jetpack Partner Portal: Introduce list of credit cards in payment method list

Context: pbtFPC-1HU-p2
Design: p6TEKc-51M-p2


#### Changes proposed in this Pull Request

* This PR loads a list of credit cards associated with a users account into the partner portal in the payment method list view.
* This is just a functional PR, and some styles may not be exact yet. We will iterate on the design shortly.
* Currently, we're loading a list of stripe credit cards associated with a user's wpcom account. This will eventually change, but that change will happen in a future iteration. This interface will work the same in either case.

![image](https://user-images.githubusercontent.com/5714212/123859091-9d744b00-d8fa-11eb-83b1-e02a3d587df7.png)


#### Testing instructions

* Load the partner portal in a dev environment and click on "Payment Methods" in the sidebar.
* You should see a list of credit cards associated with your account, along with a card for "Add new credit card".
* Ensure no console warnings/errors. Ensure that clicking on the "Add new credit card" leads to `/partner-portal/payment-method/add`. 



Related to #
